src/os/layer/preset/presetserviceaction.js

```diff
@@ -0,0 +1,16 @@
+goog.module('os.layer.preset.PresetServiceAction');
```

Comment:
Minor code style consolidation thing: both this file and settingkey.js can be consolidated into the browser preset.js utility file.

Reply:
preset.js has a circular dependency issue from staticlayerconfig, since `os/layer/config/staticlayerconfig.js` needs `os/layer/vector.js` which loops back around to layers. I'll leave the "global" in there for now and just add it to the mental list of things that'll break in layers before we can go to ES6. Wheee. <sarcasm>